apps/backend/src/auth/auth.controller.ts

```diff
@@ -47,17 +48,21 @@ export class AuthController {
 
   // TODO will be deprecated if we use Google OAuth
   @Post('/verify')
-  verifyUser(@Body() body: VerifyUserRequestDTO): void {
-    try {
-      this.authService.verifyUser(body.email, String(body.verificationCode));
-    } catch (e) {
-      throw new BadRequestException(e.message);
-    }
+  async verifyUser(@Body() body: VerifyUserRequestDTO) {
```

Comment:
❓ Since there aren't any `await` keywords in this method, we should be able to remove the `async` here, right?